x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_list/components/alerts_list.tsx

```diff
@@ -439,7 +440,7 @@ export const AlertsList: React.FunctionComponent = () => {
         }}
       >
         <AlertAdd
-          consumer={'alerts'}
+          consumer={AlertsFeatureId}
```

Comment:
This code made me realize that the change here (https://github.com/elastic/kibana/pull/66838/files?file-filters%5B%5D=.tsx#diff-11975cae1d4e0e055bfedd5944183967R442) will revoke access to alerts with a consumer of `alerting` 🤔 As in consumers of `alerting` would have to be migrated to `alerts`.

Reply:
Yeah, good point... dammit.
It's the same problem `siem` encountered when renaming to `securitySolution`.

Argh - so this is a blocker for 7.9, but not for this PR.
It should be a separate PR - I'll add an issue

Reply:
https://github.com/elastic/kibana/issues/68994

Reply:
I think in order to keep master "production-ready", we should come up with a fix for this issue before merging this PR. I'm happy to help in this area, it does have a bit of a challenge as the alerts are encrypted by the ESO plugin.

Reply:
Yeah, I'll start a separate PR.. .sigh...